sensor_msgs/JointState current_joint_state
geometry_msgs/Pose2D current_mobile_state
geometry_msgs/Pose target_ee_pose
mobile_manipulator_controller/Obstacle3D[] Obstacles3D
mobile_manipulator_controller/PoseConstraint Pose_bound
std_msgs/Bool plan_only

---
trajectory_msgs/JointTrajectory joint_trajectory
